File: VendorResponseSystem/ClientApp/src/models/work.ts
export class Work {
  public constructor(id: number, buyerCode: string, vendCode: string, vendorName: string, po: string, poitem: string, deliveryDate: string, price: string, quantity: string, isActive: boolean, isDelete: boolean, createdDate: string) {
    this.id = id;
    this.buyerCode = buyerCode;
    this.vendCode = vendCode;    
    this.po = po;
    this.poitem = poitem;
    this.deliveryDate = deliveryDate;
    this.price = price;
    this.quantity = quantity;
    this.isActive = isActive;
    this.isDelete = isDelete;
    this.createdDate = createdDate;
  }
  id!: number;
  buyerCode! : string;
  vendCode! : string;
  po! : string;
  poitem!: string;
  deliveryDate!: string;
  price!: string;
  quantity!: string;
  isActive!: boolean;
  isDelete!: boolean;
  createdDate!: string;

}

File: VendorResponseSystem/DataModels/Work.cs
using System;
using System.Collections.Generic;

namespace VendorResponseSystem.DataModels;

public partial class Work
{
    public int Id { get; set; }

    public string BuyerCode { get; set; } = null!;

    public string VendCode { get; set; } = null!;

    public decimal Po { get; set; }

    public decimal Poitem { get; set; }

    public DateTime? DeliveryDate { get; set; }

    public decimal? Quantity { get; set; }

    public decimal? Price { get; set; }

    public string? TrackingNumber { get; set; }

    public bool? IsActive { get; set; }

    public bool IsDelete { get; set; }

    public DateTime CreatedDate { get; set; }
}
